### DISCUSSION TITLE
알림 문구 변경 및 알림 센터 구현

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#210] [Features]
([#191 코멘트](https://github.com/rbgus2002/studygroup/issues/191#issuecomment-1964080848) 참고)

해당 코멘트에서 말씀해주신 부분 때문에 아래와 같이 현재 알림 메세지 문구를 정리해보았습니다.
### 공지사항
> 새로운 공지사항이 작성되었어요: {공지사항 제목}

### 댓글
> 새로운 댓글이 달렸어요: {댓글 내용}

### 과제
> {닉네임}님이 과제를 완료했어요: {과제 내용}

### 콕찌르기
> {닉네임}님이 회원님을 콕 찔렀어요 
> {닉네임}님이 회원님을 {콕찌른 횟수}번이나 콕 찔렀어요 ❗
> {닉네임}님이 회원님을 {콕찌른 횟수}번이나 콕 찔렀어요 🔥

### 과제 콕찌르기
> {닉네임}님이 회원님의 과제를 콕 찔렀어요: {과제 내용}
> {닉네임}님이 회원님의 과제를 {콕찌른 횟수}번이나 콕 찔렀어요: {과제 내용}

---
저는 다시 보니까 괜찮은 것 같기도... 한데 좀 문구가 긴 거 같기는 해요. 누가 수행했는지 직관적으로 확인을 위해 사용자의 닉네임을 넣으면 그거대로 또 길어질텐데, 이 부분은 원하시는 바가 명확하니까 생각하시는 대로 추천해주세요 ㅎㅎ

또한 현재는 알림 센터가 없어서, 알림을 한 번 확인하면 다시 확인이 불가능한 상황입니다. 따라서 **알림 센터를 만들면 어떤가** 싶습니다! 
알림 센터에는 개인이 받는 모든 알림이 기록되면 좋을 거 같아요. 디자인은 나중에 가서 뽑고, 기획적으로만 정해지면 history 기록하도록 BE에서 구현하겠습니다. 이 부분은 동의하시면 따로 discussions 만들테니 기획 정해봐요!

**Comment by rbgus2002** [comment]
예시1)
<img width="542" alt="image" src="https://github.com/rbgus2002/studygroup/assets/62997391/8758cba8-b47c-4921-b122-fa032bff3247">

**Comment by rbgus2002** [comment]
예시2) 
<img width="376" alt="image" src="https://github.com/rbgus2002/studygroup/assets/62997391/4c4abf50-81f0-4bc4-a4ea-9d1f80301fe9">

**Comment by rbgus2002** [comment]
1안) 
규규
"이슈사항 일괄 정리 및 discussions 활성화" 과제를 완료했어요

2안)
과제 완료
"이슈사항 일괄 정리 및 discussions 활성화"

**Comment by Arkady1356** [comment]
3안)
과제 완료
규규: "이슈사항 일괄 정리 및 Disccusions 활성화"

댓글
규규: "아 퇴근 시켜주세요"

공지사항
규규: "9 to 9 근무 나이스 ^^"

댓글
여섯글자이름: "매우 긴 댓글이 달렸다면 이런식으로 보여지겠지?"

... 이름 길어지니까 별로네

4안)
과제 완료 | 규규
"이슈사항 집에 가고 싶다 흑흑"

댓글 | 규규
"퇴근하고 싶다 흑흑"

공지사항 | 여섯글자이름
"이렇게 길어져도 어느정도 안정감있지롱"

5안)
규규 | 과제완료
"퇴근하고 집에 가기"


저는 4안 혹은 1안이 좋은 것 같아요

**Reply by rbgus2002** [reply]
미쳤다 4안 최곤데요 4안 고?

**Comment by rbgus2002** [comment]
4안 가시죠? 아래처럼 변경될 예정입니다.

## TOBE
### 공지사항
> 공지사항 | {닉네임}
"이렇게 길어져도 어느정도 안정감있지롱"

### 댓글
> 댓글 | {닉네임}
"퇴근하고 싶다 흑흑"

### 과제
> 과제 완료 | {닉네임}
"이슈사항 집에 가고 싶다 흑흑"

### 콕찌르기
not yet

### 과제 콕찌르기
not yet


---

콕찌르기랑 과제 콕찌르기는 좀 고민되네요. 과제 콕찌르기에 내용이 많이 들어가서 흐으음

### 콕찌르기
> 콕찌르기 | {닉네임}
"콕 찔렀어요"
"{횟수}번이나 콕 찔렀어요"

### 과제 콕찌르기
> 과제 콕찌르기 | {닉네임}
> "{과제 내용}" : 콕 찔렀어요
> "{과제 내용}" : {횟수}번이나 콕 찔렀어요

**Comment by rbgus2002** [comment]
업데이트 후 현재 아래 사진처럼 알림 오는 상태입니다~ 훨씬 깔끔한 거 같아요

<img width="381" alt="image" src="https://github.com/rbgus2002/studygroup/assets/62997391/da7550ab-6d7c-4d2c-bc1d-335ab19c5dc9">
<img width="382" alt="image" src="https://github.com/rbgus2002/studygroup/assets/62997391/86f122d7-3005-4920-ac9e-19025e85741b">
